recipes/strainr2/meta.yaml

```diff
@@ -16,10 +16,12 @@ build:
 requirements:
   build:
     - {{ compiler('c') }}
+    - zlib
```

Comment:
```suggestion
```
Build requirements are compilers and build tools like `make`, `cmake`, `m4`, etc.